Ginger/GingerCoreNET/RunLib/CLILib/CLIConfigFile.cs

```diff
@@ -37,23 +37,29 @@ public async void Execute(RunsetExecutor runsetExecutor)
         public string CreateContent(Solution solution, RunsetExecutor runsetExecutor, CLIHelper cliHelper)
         {
             string sConfig = null;
-            if (cliHelper.DownloadUpgradeSolutionFromSourceControl == true)
-            {               
-                sConfig = "SourceControlType=" + solution.SourceControl.GetSourceControlType.ToString() + Environment.NewLine;
-                sConfig += "SourceControlUrl=" + solution.SourceControl.SourceControlURL.ToString() + Environment.NewLine;
-                sConfig += "SourceControlUser=" + solution.SourceControl.SourceControlUser.ToString() + Environment.NewLine;
-                sConfig += "SourceControlPassword=" + solution.SourceControl.SourceControlPass.ToString() + Environment.NewLine;
-                if (solution.SourceControl.GetSourceControlType == SourceControlBase.eSourceControlType.GIT && solution.SourceControl.SourceControlProxyAddress.ToLower().ToString() == "true")
+            if (WorkSpace.Instance.Solution != null)
```

Comment:
we are passing solution here explicitly. So not sure why relying on Workspace.
Either to remove the method param solution if not used

Reply:
do not remove the Solution param, the dependency on Workspace was removed in porpuse, why it was added back?